app/src/views/auth/forgot_password_screen.tsx

```diff
@@ -36,7 +36,7 @@ export default function ForgotPasswordScreen({ navigation }: any) {
         />
 
         {/* Error message for the user*/}
-        {errorMessage && (
+        {Boolean(errorMessage) && (
```

Comment:
what difference does it make here ?

Reply:
Well Sonarcloud flagged this as an issue, because using Boolean(errorMessage) explicitly converts errorMessage to a true/false value, ensuring no unintended rendering if errorMessage is a truthy non-boolean value like a string or number.